style.css

```diff
@@ -0,0 +1,296 @@
+@font-face {
+  font-family: 'Lato';
+  src: url('./fonts/Lato-Regular.ttf') format('truetype');
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Lato', serif;
+}
+
+.white-link {
+  color: #fff;
+}
+
+.white-link:hover {
+  color: #f7423f;
+}
+
+.black-link {
+  color: rgba(0, 0, 0, 0.65);
+}
+
+.black-link:hover {
+  color: #f7423f;
+}
+
+.navbar {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
```

Comment:
- [ ] Lastly, As mentioned previously the navbar on the mobile version doesn't have a white background instead it should be transparent or match the background of the top section so please remove it as it's creating a discrepancy.
- Behance design:
  ![image](https://github.com/egichuhis/nakuru-snooker-tour/assets/104968236/c60bb5e5-02c0-4cf1-b75c-77ae13dc3db5)
- Your design:
  ![image](https://github.com/egichuhis/nakuru-snooker-tour/assets/104968236/2fb20691-f8b5-45b0-87a4-704c32268e60)